lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -615,17 +615,17 @@ namespace Js
                         recycler->ReportExternalMemoryFailure(length);
                     }
                 }
-                else
-                {
-                    JavascriptError::ThrowOutOfMemoryError(GetScriptContext());
-                }
             }
 
             if (buffer != nullptr)
             {
                 bufferLength = length;
                 ZeroMemory(buffer, bufferLength);
             }
+            else
```

Comment:
Akrosh, this looks like a dupe of the bug we were looking at, yes? cc @curtisman in case he knows why an ArrayBuffer buffer allocation failure doesn't OOM today